uber/site_sections/groups.py

```diff
@@ -37,7 +37,8 @@ def form(self, session, new_dealer='', first_name='', last_name='', email='', me
             message = check(group)
             if not message:
                 session.add(group)
-                message = session.assign_badges(group, params['badges'], params['badge_type'])
+                ribbon_to_use = c.NO_RIBBON if 'ribbon' not in params else params['ribbon']
```

Comment:
here's a screenshot of adding a dealer, note that 'ribbon' isn't an item here

![image](https://cloud.githubusercontent.com/assets/5413064/9142947/102eca62-3d11-11e5-9536-bcc1e0f09a0a.png)